geonetwork/content.md

```diff
@@ -12,50 +12,151 @@ The project is part of the Open Source Geospatial Foundation ( [OSGeo](http://ww
 
 %%LOGO%%
 
-# How to use this image
+## How to use this image
 
-## Start geonetwork
+GeoNetwork 4 uses an Elasticsearch server to store the index of the documents it manages so **it can't be run without configuring the URL of the Elasticsearch server**. This is a quick example of how to get GeoNetwork up and running for demo purposes. This configuration doesn't keep the data if containers are removed.
 
-This command will start a debian-based container, running a Tomcat web server, with a geonetwork war deployed on the server:
+```console
+docker pull docker.elastic.co/elasticsearch/elasticsearch:7.17.15
+docker pull geonetwork:4
+
+docker network create gn-network
+
+docker run -d --name my-es-host --network gn-network -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.17.15
+docker run --name geonetwork-host --network gn-network -e GN_CONFIG_PROPERTIES="-Des.host=my-es-host -Des.protocol=http -Des.port=9200 -Des.url=http://my-es-host:9200" -p 8080:8080 geonetwork:4
+```
+
+For GeoNetwork 4.2:
 
 ```console
-$ docker run --name some-%%REPO%% -d %%IMAGE%%
+docker network create gn-network
```

Comment:
```suggestion
docker pull docker.elastic.co/elasticsearch/elasticsearch:7.17.15
docker pull geonetwork:4.2

docker network create gn-network
```

Reply:
Adding the docker pull so I can get a single cut and paste.

Reply:
References here to `geonetwork` should be `%%IMAGE%%` instead so they get templated appropriately.

Also, references to `elasticsearch` should be to https://hub.docker.com/_/elasticsearch instead

Reply:
@tianon replaced `geonetwork` by `%%IMAGE%% and the references to elasticsearch by `elasticsearch:7.17.15`